Plugins~/Src/MeshSyncClientBlender/python/3.1.0/unity_mesh_sync.py

```diff
@@ -128,6 +128,12 @@ def __del__(self):
     def execute(self, context):
         return self.invoke(context, None)
 
+    # When a file is loaded, the modal registration is reset:
```

Comment:
We should also apply these changes to other `unity_mesh_sync.py` files

Reply:
Done